docs/data/routes/docs/nextjs/data-fetching/data-services/en.md

```diff
@@ -3,4 +3,71 @@ name: data-services
+
+The method returns an object of the type `SitecorePageProps`.
+
+```tsx
+export type SitecorePageProps = {
+  locale: string;
+  layoutData: LayoutServiceData | null;
+  dictionary: DictionaryPhrases;
+  componentProps: ComponentPropsCollection;
+  notFound: boolean;
+};
+```
+
+ See the full definition of `SitecorePageProps` in [`src/lib/page-props.ts`](https://github.com/Sitecore/jss/blob/master/samples/nextjs/src/lib/page-props.ts).
+
+## Page `props`
+
+To prepare the page `props`, the `SitecorePagePropsFactory` uses the [Layout Service](/docs/fundamentals/services/layout-service).and the [Dictionary Service](/docs/fundamentals/services/dictionary-service). 
+
+> If you need additional data for every page, you should fetch it inside the `SitecorePagePropsFactory` and return it together with the other properties.
+
+> We are currently using standard REST-based dictionary and layout services. For Sitecore Experience Edge, we will soon provide GraphQL-based implementations.
+
+### locale
+
+The factory retrieves the `context` locale as configured in the `i18n` entry of `next.config.js`. Otherwise, it uses the  `config.language` defined in `package.json`.
+
+### layoutData
+
+The `layoutData` prop stores `LayoutServiceData` from the Sitecore Layout Service provided by a `RestLayoutService` instance. The `RestLayoutServices` fetches `LayoutServiceData` using the Sitecore Layout Service REST API using Axios as the default data fetcher. In SSR mode, it will set up the required request and response `headers` using the attributes `context.req` and `context.res` to provide the ability to use [Sitecore Tracking](/docs/fundamentals/services/tracking#jss-tracking).
+
+### dictionary
+
+The `dictionary` prop contains `DictionaryPhrases` from the Sitecore Dictionary Service. A `RestDictionaryService` instance fetches dictionary data for the given language using the Sitecore Dictionary Service REST API. The `RestDictionaryService` uses Axios as the default data fetcher. By default, `caching` is enabled, and the `cacheTimeout` is 60 sec. You can disable/enable caching using the `cacheEnabled` property, and you can customize cache timeout using the `cacheTimeout` property.
+
+### componentProps
+
+The `componentProps` contain data fetched by `ComponentPropsService`. The `ComponentPropsService` executes `getStaticProps/getServerSideProp` in those components that implement these functions. `componentProps` is a key-value storage of the format `{ [renderingUID]: data }`. See this [example](https://github.com/Sitecore/jss/blob/master/samples/nextjs/src/components/graphql/GraphQL-ConnectedDemo.tsx).
+
+### notFound
+
+If the Layout Service returns a `404` status for our page route, it will update the value of `notFound` to `true`, triggering our custom [NotFound](https://github.com/Sitecore/jss/blob/master/samples/nextjs/src/components/NotFound.tsx) page with the correct `404` status. 
+
+## Experience Editor (Preview)
```

Comment:
This "Preview" might make people think that the Experience Editor functionality is in release preview? Maybe instead "Experience Editor (Next.js Preview Context)"